.github/workflows/wordpress.yml

```
@@ -54,7 +54,7 @@ jobs:
 
       - name: Setup tests
         run: |
-            bash tests/bin/install.sh wordpress_test root root localhost ${{ matrix.wordpress }}
+            bash tests/unit/bin/install.sh wordpress_test root root localhost ${{ matrix.wordpress }}
 
       -   name: Execute tests
           run: vendor/bin/phpunit
```

@kjohnson Just because I happened to run into this on GiveIO, I suggest the following:
```suggestion
          run: php -d memory_limit=-1 vendor/bin/phpunit
```

---

Sounds good! Suggestion merged.

For reference: https://github.com/impress-org/giveio/commit/b401c1a4dc794877a4e3362b544d06408fc5fed0